qfieldsync/core/cloud_api.py

```diff
@@ -270,19 +395,52 @@ def login(self, username: str, password: str) -> Optional[QNetworkReply]:
                 "password": password,
             },
         )
-        reply.finished.connect(lambda: self._on_login_finished(reply))
+        reply.finished.connect(lambda: self._on_login_with_credentials_finished(reply))
+
+        return reply
+
+    def _get_cloud_user_info(self) -> Optional[QNetworkReply]:
```

Comment:
Would you mind moving the private methods somewhere in the bottom or at least not splitting the two important methods `login_with_...` in two.